### PR TITLE
Splunk py raise timezone exception

### PR DESCRIPTION
**Pull request by darkushin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-24965

## Description
Fixed an issue where the returned error message when defining mirroring without specifying a timezone wasn't informative;

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Review by ilappe — CHANGES_REQUESTED** [review]
Nice job!

File: Packs/SplunkPy/Integrations/SplunkPy/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This integration was integrated and tested with Splunk v7.2.
     | Use Splunk Clock Time For Fetch |  | False |
     | Parse Raw Part of Notable Events |  | False |
     | Replace with Underscore in Incident Fields |  | False |
-    | Timezone of the Splunk server, in minutes. For example, if GMT is gmt +3, set timezone to +180. For UTC, set the timezone to 0. (Set only if the Splunk server is different than the Cortex XSOAR server.) Relevant only for fetching and mirroring notable events. |  | False |
+    | Timezone of the Splunk server, in minutes. For example, if GMT is gmt +3, set timezone to +180. For UTC, set the timezone to 0. Relevant only for fetching and mirroring notable events. Most be specified when mirroring is enabled. |  | False |
```
Comment (yuvalbenshalom) [inline Packs/SplunkPy/Integrations/SplunkPy/README.md:33]:
```suggestion
    | Timezone of the Splunk server, in minutes. For example, if GMT is gmt +3, set timezone to +180. For UTC, set the timezone to 0. This is relevant only for fetching and mirroring notable events. It Must be specified when mirroring is enabled. |  | False |
```

File: Packs/SplunkPy/ReleaseNotes/3_0_20.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### SplunkPy
+
+- Fixed an issue where the returned error message when enabling mirroring without specifying a timezone was not informative.
```
Comment (yuvalbenshalom) [inline Packs/SplunkPy/ReleaseNotes/3_0_20.md:6]:
```suggestion
- Fixed an issue where the it was possible to configure mirroring without specifying a timezone.
```

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml [hunk 3]
```diff
@@ -65,7 +65,7 @@ configuration:
   type: 8
   section: Collect
   advanced: true
-- display: Timezone of the Splunk server, in minutes. For example, if GMT is gmt +3, set timezone to +180. For UTC, set the timezone to 0. (Set only if the Splunk server is different than the Cortex XSOAR server.) Relevant only for fetching and mirroring notable events.
+- display: Timezone of the Splunk server, in minutes. For example, if GMT is gmt +3, set timezone to +180. For UTC, set the timezone to 0. Relevant only for fetching and mirroring notable events. Most be specified when mirroring is enabled.
```
Comment (yuvalbenshalom) [inline Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml:68]:
```suggestion
- display: Timezone of the Splunk server, in minutes. For example, if GMT is gmt +3, set timezone to +180. For UTC, set the timezone to 0. This is relevant only for fetching and mirroring notable events. It Must be specified when mirroring is enabled.
```

**Review by yuvalbenshalom — APPROVED** [review]

**Review by ilappe — APPROVED** [review]

**Pull request merged**